.gitignore

```diff
@@ -2,9 +2,14 @@
 # will have compiled files and executables
 /target/
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# Before:
+# > Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# After:
+# > Always place Cargo.lock into git repository.
+
 # More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
+# [Why have Cargo.lock in version control?](https://doc.rust-lang.org/cargo/faq.html#why-have-cargolock-in-version-control)
+# Cargo.lock
```

Comment:
Please remove the lines and below (line 5-12 and 22-28) for `Cargo.lock`. Leaving discuss notes about accepting it may unnecessary.